java/pom.xml

```diff
@@ -0,0 +1,125 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Maybe we can put our code in `mlflow/java/`, to follow [tensorflow's pattern](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/java)